packages/frontend-2/lib/viewer/composables/setup/selection.ts

```diff
@@ -45,6 +46,11 @@ function useSelectOrZoomOnSelection() {
         if (args.hits.length === 0) return trackAndClearSelection()
         if (!args.multiple) clearSelection() // note we're not tracking selectino clearing here
 
+        // Skip if selection disabled
+        if (!state.viewer.instance.getExtension(SelectionExtension).enabled) {
```

Comment:
@AlexandruPopovici is this a viewer bug? selection events still keep firing even tho the extension is disabled